x-pack/plugins/fleet/server/routes/app/index.ts

```diff
@@ -98,6 +104,49 @@ export const getCheckPermissionsHandler: FleetRequestHandler<
   }
 };
 
+export const postEnableSpaceAwarenessHandler: FleetRequestHandler = async (
+  context,
+  request,
+  response
+) => {
+  try {
+    const soClient = appContextService.getInternalUserSOClientWithoutSpaceExtension();
+
+    // TODO If settings SO already set => return
+
+    // Migration
+    // For every policy => create a new one (shortcut for POC do not write package policies)
+    const res = await soClient.find<any>({
+      type: LEGACY_AGENT_POLICY_SAVED_OBJECT_TYPE,
+      perPage: SO_SEARCH_LIMIT,
+    });
+
+    // Should probably check results
+    await soClient.bulkCreate<any>(
```

Comment:
For me it looks like the migration from agnostic to => multiple is as simple as creating the existing saved object in the new saved objects